### PR TITLE
Migration and update seeds.

### PR DESCRIPTION
**Pull request by pedrosino**

### CODE REVIEW

**Comment by pedrosino** [comment]
Screenshot:
events

File: WcaOnRails/db/seeds/events.seeds.rb [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
-('444', '4x4 Cube', 20, 'time', '4x4 Cube'),
-('555', '5x5 Cube', 30, 'time', '5x5 Cube'),
-('222', '2x2 Cube', 40, 'time', '2x2 Cube'),
-('333bf', 'Rubik''s Cube: Blindfolded', 50, 'time', '3x3 blindfolded'),
-('333oh', 'Rubik''s Cube: One-handed', 60, 'time', '3x3 one-handed'),
-('333fm', 'Rubik''s Cube: Fewest moves', 70, 'number', '3x3 fewest moves'),
-('333ft', 'Rubik''s Cube: With feet', 80, 'time', '3x3 with feet'),
+('444', '4x4x4 Cube', 30, 'time', '4x4x4 Cube'),
+('555', '5x5x5 Cube', 40, 'time', '5x5x5 Cube'),
+('222', '2x2x2 Cube', 20, 'time', '2x2x2 Cube'),
```
Comment (jfly) [inline WcaOnRails/db/seeds/events.seeds.rb]: Could you reorder these by rank?

File: WcaOnRails/db/migrate/20161226223701_change_event_order_and_names.rb [hunk 2]
```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+class ChangeEventOrderAndNames < ActiveRecord::Migration
+  def up
+    # No change for Rubik's Cube event.
```
Comment (jfly) [inline WcaOnRails/db/migrate/20161226223701_change_event_order_and_names.rb]: Just to make it clear, how about we do one line at  the end where we set `cellName = name`?

Comment (pedrosino) [inline WcaOnRails/db/migrate/20161226223701_change_event_order_and_names.rb]: Ah, that's smart.

**Comment by pedrosino** [comment]
I have fixed @jfly comments. I also pushed this to staging and the dropdown menu looks good:
eventsdrop

Since rubocop was complaining about the test line being too long (251x245) I changed Rubocop's policy to 255. Let me know if there's a better way of dealing with this.

File: WcaOnRails/db/migrate/20161226223701_change_event_order_and_names.rb [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    SQL
+    execute <<-SQL
+      update Events set name = '5x5x5 Blindfolded', rank = 170 where id = '555bf';
+    SQL
+    execute <<-SQL
+      update Events set name = '3x3x3 Multi-Blind', rank = 180 where id = '333mbf';
+    SQL
+
+    execute <<-SQL
+      update Events set cellName = name;
```
Comment (jfly) [inline WcaOnRails/db/migrate/20161226223701_change_event_order_and_names.rb]: Looks good. Could you create an issue to drop the `cellName` column and assign it to yourself? I don't want to take care of it in this PR, but I also would like to take care of it sooner than later.

File: WcaOnRails/spec/controllers/competitions_controller_spec.rb [hunk 4]
```diff
@@ -597,8 +597,8 @@
         expect(competition.results_posted_at).to be nil
         get :post_results, id: competition
         post = assigns(:post)
-        expect(post.body).to include "World records: Jeremy Fleischman 3x3 one-handed 50.00 (average), Vincent Sheu (2006SHEU01) 3x3 fewest moves 25 (single), 3x3 fewest moves 26.00 (average), Vincent Sheu (2006SHEU02) 2x2 Cube 10.00 (single)"
-        expect(post.body).to include "North American records: Jeremy Fleischman 3x3 one-handed 41.00 (single), 3x3 one-handed 40.00 (single)"
+        expect(post.body).to include "World records: Jeremy Fleischman 3x3x3 One Handed 50.00 (average), Vincent Sheu (2006SHEU01) 3x3x3 Fewest Moves 25 (single), 3x3x3 Fewest Moves 26.00 (average), Vincent Sheu (2006SHEU02) 2x2x2 Cube 10.00 (single)"
```
Comment (jfly) [inline WcaOnRails/spec/controllers/competitions_controller_spec.rb]: So this is our longest line of Ruby code? I'd say it's time to split it up into multiple lines. I like to write multiline strings like this:

```ruby
expect(post.body).to include(
  "World records: Jeremy Fleischman 3x3x3 One Handed 50.00 (average), " +
  "Vincent Sheu (2006SHEU01) 3x3x3 Fewest Moves 25 (single), " +
  "3x3x3 Fewest Moves 26.00 (average), " +
  "Vincent Sheu (2006SHEU02) 2x2x2 Cube 10.00 (single)"
)
```

File: WcaOnRails/db/migrate/20161226223701_change_event_order_and_names.rb (hunk 3 above)
Comment (pedrosino) [inline WcaOnRails/db/migrate/20161226223701_change_event_order_and_names.rb]: #1054

File: WcaOnRails/spec/controllers/competitions_controller_spec.rb (hunk 4 above)
Comment (pedrosino) [inline WcaOnRails/spec/controllers/competitions_controller_spec.rb]: Rubocop says we should use \ instead of +.

**Comment by jfly** [comment]
I'm trusting you to check that our seeds match the migration. In the future, you could avoid this worry by having the migration drop the events table and load from seeds.

LGTM.

**Pull request merged**